globalid.gemspec

```diff
@@ -16,7 +16,10 @@ Gem::Specification.new do |s|
   s.files        = Dir['MIT-LICENSE', 'README.md', 'lib/**/*']
   s.require_path = 'lib'
 
-  s.add_runtime_dependency 'activesupport', '>= 4.2.0'
+  s.add_runtime_dependency 'activesupport', ['>= 3.2.0', '< 4.2']
 
   s.add_development_dependency 'rake'
+  s.add_development_dependency 'minitest', '< 5.11'
```

Comment:
Minitest 5.11 apparently includes a change that breaks some old tests (e.g. https://github.com/rails/rails-perftest/issues/43).  This was supposedly patched (https://github.com/rails-api/active_model_serializers/pull/2289), but seems not to be playing nicely with Rails 4.  Downgrading minitest should work for now.